modules/oadp-release-notes-1-2-3.adoc

```diff
@@ -34,5 +34,10 @@ For a complete list of all issues resolved in the release of OADP 1.2.3, see the
 [id="known-issues-1-2-3_{context}"]
 == Known issues
 
-There are no known issues in the release of OADP 1.2.3.
+The {oadp-short} 1.2.3 has the following known issue:
 
+.DPA does not reconcile when the credentials secret is updated
```

Comment:
Please expand the _DPA_ acronym on the first usage.

See the [IBM Style (ISG)](https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations#spelled-out-forms) rule to spell out abbreviations for the first use.